drivers/src/hand_off.rs

```diff
@@ -289,7 +289,19 @@ pub struct FirmwareHandoffTable {
     /// RtAlias TBS Size.
     pub rtalias_tbs_size: u16,
 
+    /// Maximum value RT FW SVN can take.
+    #[cfg(any(feature = "fmc", feature = "runtime"))]
+    pub rt_hash_chain_max_svn: u16,
+
+    /// Index of RT hash chain value in the Key Vault.
+    #[cfg(any(feature = "fmc", feature = "runtime"))]
+    pub rt_hash_chain_kv_hdl: HandOffDataHandle,
+
     /// Reserved for future use.
+    #[cfg(any(feature = "fmc", feature = "runtime"))]
+    pub reserved: [u8; 1636],
+
+    #[cfg(not(any(feature = "fmc", feature = "runtime")))]
```

Comment:
I think it would be better to have these fields defined always, not based on configuration. I feel like it will get tricky to manage if the structure starts looking different depending on how the FW was built.

Reply:
I'd like to always define these as well, it's just that without these cfg-guards the ROM binary changes.

Reply:
ahh, got it.